frontend/src/utils/frontend-config-fetcher.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { getConfig } from '../api/config'
+import type { FrontendConfig } from '../api/config/types'
+import type { BaseUrls } from '../components/common/base-url/base-url-context-provider'
+import { Logger } from './logger'
+
+/**
+ * Fetches and caches the {@link FrontendConfig frontend config} from the backend.
+ */
+export class FrontendConfigFetcher {
+  private readonly logger = new Logger('Frontend config')
```

Comment:
Spoken exactly, this logs messages for the frontend config fetcher and not the config itself.

```suggestion
  private readonly logger = new Logger('Frontend config fetcher')
```